releases/release-4.0.11.md

```diff
@@ -0,0 +1,169 @@
+    - 当 JSON 数据长度超过 65536 时提供报错 [#21870](https://github.com/pingcap/tidb/pull/21870)
+    - 修复 `dyname` 函数和 MySQL 不兼容的问题 [#21850](https://github.com/pingcap/tidb/pull/21850)
+    - 修复输入数据过长时 `to_base64` 函数返回 `NULL` 的问题 [#21813](https://github.com/pingcap/tidb/pull/21813)
+    - 修复在子查询中比较多个字段失败的问题 [#21808](https://github.com/pingcap/tidb/pull/21808)
+    - 修复 JSON 中比较浮点数的问题 [#21785](https://github.com/pingcap/tidb/pull/21785)
+    - 修复 JSON 类型比较的问题 [#21718](https://github.com/pingcap/tidb/pull/21718)
+    - 修复 `cast` 函数的 coercibility 值设置错误的问题 [#21714](https://github.com/pingcap/tidb/pull/21714)
+    - 修复使用 `IF` 函数时可能出现 panic 的问题 [#21711](https://github.com/pingcap/tidb/pull/21711)
+    - 修复 JSON 搜索返回 `NULL` 和 MySQL 不兼容的问题 [#21700](https://github.com/pingcap/tidb/pull/21700)
+    - 修复 `ORDER BY` 和 `HAVING` 子句检查 `only_full_group_by` 模式的问题 [#21697](https://github.com/pingcap/tidb/pull/21697)
+    - 修复 Day/Time 单位和 MySQL 不兼容的问题 [#21676](https://github.com/pingcap/tidb/pull/21676)
+    - 修复 `LEAD` 和 `LAG` 函数默认值类型问题 [#21665](https://github.com/pingcap/tidb/pull/21665)
+    - `LOAD DATA` 时执行检测以保证只能往基础表中导入数据 [#21638](https://github.com/pingcap/tidb/pull/21638)
+    - 修复 `addtime` 和 `subtime` 函数处理非法参数的问题 [#21635](https://github.com/pingcap/tidb/pull/21635)
+    - 将近似值的舍入规则更改为“舍入到最接近的偶数” [#21628](https://github.com/pingcap/tidb/pull/21628)
+    - 修复 `WEEK()` 在被明确读取前无法识别全局变量 `default_week_format` 的问题 [#21623](https://github.com/pingcap/tidb/pull/21623)
+
++ TiKV
+
+    - 修复当设置 `PROST=1` 时构建 TiKV 失败的问题 [#9604](https://github.com/tikv/tikv/pull/9604)
+    - 修复不匹配的内存诊断信息 [#9589](https://github.com/tikv/tikv/pull/9589)
+    - 修复在恢复 RawKV 数据时部分 key range 的 end key 的包含性问题 [#9583](https://github.com/tikv/tikv/pull/9583)
+    - 修复当 TiCDC 增量扫数据时读取一个被回滚的事务的某个 key 的旧值时 TiKV 可能会 panic 的问题 [#9569](https://github.com/tikv/tikv/pull/9569)
+    - 修复使用不同配置的连接拉取同一个 Region 的变更时旧值配置不匹配的问题 [#9565](https://github.com/tikv/tikv/pull/9565)
+    - 修复 TiVK 运行在网络接口缺少 MAC 地址的设备上会崩溃的问题（自 v4.0.9 引入）[#9516](https://github.com/tikv/tikv/pull/9516)
+    - 修复 TiKV 在备份大 Region 时会内存溢出的问题 [#9448](https://github.com/tikv/tikv/pull/9448)
+    - 修复 `region-split-check-diff` 无法自定义配置的问题 [#9530](https://github.com/tikv/tikv/pull/9530)
+    - 修复系统时间回退时 TiKV 会 panic 的问题 [#9542](https://github.com/tikv/tikv/pull/9542)
+
++ PD
+
+    - 修复成员健康的监控显示不正确的问题 [#3368](https://github.com/pingcap/pd/pull/3368)
+    - 禁止有副本的不正常 tombstone store 被清除 [#3352](https://github.com/pingcap/pd/pull/3352)
+    - 修复 store limit 无法持久化的问题 [#3403](https://github.com/pingcap/pd/pull/3403)
+    - 调整 `scatter range schedler` 的 limit 限制 [#3401](https://github.com/pingcap/pd/pull/3401)
+
++ TiFlash
+
+    - 修正 Decimal 类型的 `min`/`max` 计算结果错误的问题
+    - 修正读取数据时有可能导致 crash 的问题 [#1358](https://github.com/pingcap/tics/pull/1358)
+    - 修正 DDL 操作后写入的数据可能会在 compaction 后丢失的问题
+    - 修正 Coprocessor 中错误解析 Decimal 常量的问题
+    - 修正 Learner Read 过程中可能导致 crash 的问题
+    - 修正 TiFlash 中除 `0` 或 `NULL` 的行为与 TiDB 不一致的问题
```

Comment:
line 140 to line 143, No PR link is provided after these items.

Reply:
TiFlash is not open-sourced, so TiFlash notes have no PRs appended.

Reply:
> TiFlash is not open-sourced, so TiFlash notes have no PRs appended.

OK. Thanks for the explanation. Please ignore the comment.